### DISCUSSION TITLE
Skills Bugs

### DISCUSSION THREAD

**Discussion by NeonDaniel** [NeonGeckoCom/NeonCore#334] [General]
This thread will be updated with links to known skill issues to track their status in one place. Feel free to comment on this thread with any newly discovered skill issues (bonus points for linking to an issue in the appropriate skill repository).

**Comment by NeonDaniel** [comment]
Fix the 'show pictures' intent
https://github.com/NeonGeckoCom/skill-camera/issues/29

**Comment by NeonDaniel** [comment]
Repair and re-enable Audio Recordings Skill
https://github.com/NeonGeckoCom/skill-audio_record/issues/15
https://github.com/NeonGeckoCom/skill-audio_record/issues/16

The audio recording skill has been blacklisted in Neon images for now until these bugs can be addressed

**Comment by NeonDaniel** [comment]
Launcher skill spoken errors on Mark2
https://github.com/NeonGeckoCom/skill-launcher/issues/28
https://github.com/NeonGeckoCom/neon-image-recipe/issues/70

**Reply by NeonDaniel** [reply]
Patched to prevent spoken errors, just need to validate the fix: https://github.com/NeonGeckoCom/NeonCore/pull/340

**Comment by NeonDaniel** [comment]
Stock skill not handling intents like "what is amazon trading at"
https://github.com/NeonGeckoCom/skill-stock/issues/18

**Reply by NeonDaniel** [reply]
This requires some better parsing of company names from utterances and probably a refactoring to be a CommonQuery skill. For now, the invalid intent has been removed from examples: https://github.com/NeonGeckoCom/skill-stock/pull/27

**Reply by NeonDaniel** [reply]
Intents Updated with tests https://github.com/NeonGeckoCom/skill-stock/pull/35

**Comment by NeonDaniel** [comment]
Homescreen doesn't reflect user time format changes
https://github.com/NeonGeckoCom/skill-user_settings/issues/53

**Reply by NeonDaniel** [reply]
Issue transferred and closed, just need to validate the fix: https://github.com/NeonGeckoCom/NeonCore/pull/340

**Comment by NeonDaniel** [comment]
Asking for username prompts the user to add a username, even if it is unsupported on a particular platform
https://github.com/NeonGeckoCom/skill-user_settings/issues/54

**Reply by NeonDaniel** [reply]
Issue closed, just need to validate the fix: https://github.com/NeonGeckoCom/NeonCore/pull/340

**Comment by NeonDaniel** [comment]
News skill not handling intents in latest release (22.10.3)

**Reply by NeonDaniel** [reply]
Testing [this PR](https://github.com/NeonGeckoCom/NeonCore/pull/368), I do get news (Sky News) if I ask `play the latest news`, but I don't get NPR news if I ask `play npr news` @JarbasAl

**Comment by NeonDaniel** [comment]
Homescreen weather sometimes does not reflect current condition (as reported by skill intent).

Reported [in Matrix](https://matrix.to/#/!ZhEZYNzKBfpEAhtQIz:matrix.org/$zavvJg9yvz898wwG-Ye5jS9oVTVQwYvONT61xLsnaLc?via=matrix.org&via=kde.org&via=halogen.city)

**Reply by NeonDaniel** [reply]
Confirmed by multiple user reports. Solution will likely come from moving to the OVOS weather skill

**Reply by NeonDaniel** [reply]
OVOS skill appears to use similar or the same logic as Neon; this may have been related to https://github.com/NeonGeckoCom/neon-utils/pull/438

**Comment by RedPathWalker** [comment]
Am I correct in understanding that this is where you'd like skill problems reported for Neon AI Mycroft Mark II skills.  If so here is the link to my report of an issue that I reported via the matrix system.  This is getting very confusing as I've spent the last couple days trying to catch up with Neon AI for Mycroft Mark II only to be told to take my reports of issues here.  I don't want to be a problem just a help.  Link follows: https://matrix.to/#/!RStqiRzLiVgGQCEXei:matrix.org/$ait730c76ZHNsYnVIIVSZB7eOnFZEecN3MMU2QkJtrs

**Reply by NeonDaniel** [reply]
This is a good place to track bugs since resolutions can be linked back to the discussion; Matrix will get faster feedback and is better for questions. The idea is that this page can be referenced to see any known skill bugs in one place.

I've opened [an issue](https://github.com/NeonGeckoCom/skill-update/issues/44) for this bug and will post an update on this thread when a fix is implemented

**Comment by RedPathWalker** [comment]
I am lacking clarity.  I note that the destination your link provided to
report problems has a number 344 attached to it.  Did you mean that I
should report problems to this specific destination or did you mean the
topic in general?  If the latter is there a specific link that would be a
landing place I should be using?  Is there a specific procedure I should
follow to leave you issues I discover/encounter?

Gayland
KG7GCF
